core/src/types/capability.rs

```diff
@@ -193,7 +193,7 @@ pub struct Capability {
     /// Indicates whether last modified timestamp is available in list response
     pub list_has_last_modified: bool,
     /// Indicates whether version information is available in list response
-    pub list_has_version: bool,
+    pub list_has_versioned: bool,
```

Comment:
This doesn't need to change. the version is the `version` in `metadata`, not for the `versioned` filter in `list` operation.